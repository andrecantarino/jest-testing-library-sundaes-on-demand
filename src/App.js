import "./App.css";
import SummaryForm from "./pages/summary/SummaryForm";
import Options from "./pages/entry/Options";

function App() {
  return (
    <div>
      <Options />
      <SummaryForm />
    </div>
  );
}

export default App;
