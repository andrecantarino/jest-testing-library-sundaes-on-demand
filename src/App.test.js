import { render, screen } from "@testing-library/react";
import App from "./App";
import userEvent from "@testing-library/user-event";

test("should get a snapshot", () => {
  const { asFragment } = render(<App />);

  expect(asFragment(<App />)).toMatchSnapshot();
});
