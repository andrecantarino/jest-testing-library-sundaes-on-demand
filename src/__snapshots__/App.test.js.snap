// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should get a snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <div>
      <h2>
        Scoops
      </h2>
      <p>
        2 each
      </p>
      <p>
        Scoops total: $0.00
      </p>
      <div
        class="row"
      />
      <h2>
        Toppings
      </h2>
      <p>
        1.5 each
      </p>
      <p>
        Toppings total: $0.00
      </p>
      <div
        class="row"
      />
    </div>
  </div>
</DocumentFragment>
`;
